cirq/circuits/circuit.py

```diff
@@ -659,6 +660,50 @@ def findall_operations_between(self,
 
         return list(result)
 
+    def findall_operations_until_blocked(self,
+                                   start_frontier: Dict[ops.QubitId, int],
+                                   is_blocker: Callable[[ops.Operation], bool]
+                                         = lambda op: False,
+                                   omit_crossing_operations: bool = False
+                                   ) -> List[Tuple[int, ops.Operation]]:
+        """
+        Finds all operations until a blocking operation is hit.  For each
+        qubit, this returns a list of all operations that involve that qubit
```

Comment:
This sentence makes it sound like the return type should be `Dict[Qid, List[...]]`. Rephrase to make it clearer how the lists are aggregated.